README.rst

```diff
@@ -425,6 +428,135 @@ view (only when using HTTP POST).
 The signal is emitted just before a successful response is returned,
 it is not sent if the response was not successful.
 
+Default Metrics
+---------------
+
+``Ping``
+~~~~~~~~
+
++--------------+-------------------------------------------------+
+|     name     |              ``Ping <Device>``                  |
++--------------+-------------------------------------------------+
+|     key      |                   ``ping``                      |
```

Comment:
What about saying "Measurement" instead of just key? Key doesn't tell me anything at all. Measurement is more tied to influxdb but we can find a better name when we finish abstracting the metrics from the db

Reply:
Well, I read that it is simply called `metric` in the case of `Prometheus` and several other timeseries db. I think giving this field the name `measurement/metric` might confuse new users unless they are acquainted with influxdb and how metric is acting as a middle agent here (less likely, I think). Suppose they are creating a new metric they will find the field `key` to fill and not `measurement` (increases confusion in my opinion but you might know better).

Reply:
IMO as a end-user or potential customer of openwisp-monitoring I expect the README to describe the metrics that I will get for free when using this module, rather than technical details of the Metric model, which is not useful to evaluate if I want to use the module or not and after all is going to change when we continue refactoring it